media-autobuild_suite.bat

```diff
@@ -170,6 +170,14 @@ endlocal & set deleteIni=%deleteIni%
 
 if %deleteINI%==1 echo.[compiler list] >"%ini%"
 
+:ffmpegPath
+set "writePath=no"
+if %ffmpegPathINI%==0 (
+    set ffmpegPath="https://git.ffmpeg.org/ffmpeg.git"
+    ) else set ffmpegPath=%ffmpegPathINI%
+if %deleteINI%==1 set "writePath=yes"
+if %writePath%==yes echo.ffmpegPath=%ffmpegPath%>>%ini%
```

Comment:
```suggestion
if %deleteINI%==1 echo.ffmpegPath=%ffmpegPath%>>%ini%
```
There doesn't seem to be an obvious reason to have the writePath variable here?

Reply:
I believe that in a very early version I had tried to make this part of the interactive questioning procedure but later got away from doing that.